src/theme/colors/index.ts

```diff
@@ -0,0 +1,144 @@
+import * as localContrasts from './contrasts'
+
+export type ContrastType = {
+  100: string
+  200: string
+  300: string
+  400: string
+  500: string
+  600: string
+  700: string
+  800: string
+  900: string
+  1000?: string
+}
+
+type GenerateTokensProps = {
+  sentiment: string
+  contrast: ContrastType
+  neutralContrast: ContrastType
+}
+
+// This function will generate all colors tokens using shade of contrasts
+const generateTokens = ({
+  sentiment,
+  contrast,
+  neutralContrast,
+}: GenerateTokensProps) => {
+  // Neutral is a particular color it has more shade and they are used differently than usual colors.
+  if (sentiment === 'neutral') {
+    return {
+      /* eslint-disable sort-keys */
+
+      // Background
+      background: contrast[400],
+      backgroundHover: contrast[400],
+      backgroundDisabled: contrast[400],
+      backgroundWeak: contrast[100],
+      backgroundWeakHover: contrast[800],
+      backgroundWeakDisabled: contrast[100],
+      backgroundStrong: contrast[800],
+      backgroundStrongHover: contrast[800],
+      backgroundStrongDisabled: contrast[800],
+
+      // Text
+      text: contrast[800],
+      textHover: contrast[800],
+      textDisabled: contrast[800],
+      textWeak: contrast[500],
+      textWeakHover: contrast[100],
+      textWeakDisabled: contrast[500],
+      textStrong: contrast[100],
+      textStrongHover: contrast[100],
+      textStrongDisabled: contrast[100],
+
+      // Border
+      border: contrast[800],
+      borderHover: contrast[800],
+      borderDisabled: contrast[800],
+      borderWeak: contrast[800],
+      borderWeakHover: contrast[800],
+      borderWeakDisabled: contrast[800],
+      borderStrong: contrast[800],
+      borderStrongHover: contrast[800],
+      borderStrongDisabled: contrast[800],
+
+      /* eslint-enable */
+    }
+  }
+
+  return {
+    /* eslint-disable sort-keys */
+
+    // Background
+    background: contrast[100],
+    backgroundHover: contrast[100],
+    backgroundDisabled: contrast[100],
+    backgroundWeak: neutralContrast[100],
+    backgroundWeakHover: contrast[800],
+    backgroundWeakDisabled: neutralContrast[100],
+    backgroundStrong: contrast[800],
+    backgroundStrongHover: contrast[800],
+    backgroundStrongDisabled: contrast[800],
+
+    // Text
+    text: contrast[800],
+    textHover: contrast[800],
+    textDisabled: contrast[800],
+    textWeak: contrast[800],
+    textWeakHover: contrast[100],
+    textWeakDisabled: contrast[800],
+    textStrong: neutralContrast[100],
+    textStrongHover: neutralContrast[100],
+    textStrongDisabled: neutralContrast[100],
+
+    // Border
+    border: contrast[800],
+    borderHover: contrast[800],
+    borderDisabled: contrast[800],
+    borderWeak: contrast[800],
+    borderWeakHover: contrast[800],
+    borderWeakDisabled: contrast[800],
+    borderStrong: contrast[800],
+    borderStrongHover: contrast[800],
+    borderStrongDisabled: contrast[800],
+
+    /* eslint-enable */
```

Comment:
```suggestion
    /* eslint-enable sort-keys */
```

Might be a good idea to turn off this rule once and for all 

@philibea do you still find any value in this rule ?

Reply:
This rule was for propTypes and props in general if I remember well

Reply:
There is a dedicated rule for propTypes https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md